From mkatiyar@gmail.com  Fri Jan 11 11:17:23 2008
From: "Manish Katiyar" <mkatiyar@gmail.com>
Date: Sat, 15 Dec 2007 11:01:14 +0530
Subject: [linux-usb-devel] [PATCH 2.6.23.11] USB : correct comments in usb/core/notify.c
To: linux-usb-devel@lists.sourceforge.net
Message-ID: <ea11fea30712142131v223cc3e5s7d180d97d4aabd1e@mail.gmail.com>
Content-Disposition: inline


This patch corrects the wrong function name mentioned in the comments
of usb_unregister_notify function.

Signed-off-by: Manish Katiyar <mkatiyar@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/notify.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/core/notify.c
+++ b/drivers/usb/core/notify.c
@@ -33,7 +33,7 @@ EXPORT_SYMBOL_GPL(usb_register_notify);
  * usb_unregister_notify - unregister a notifier callback
  * @nb: pointer to the notifier block for the callback events.
  *
- * usb_register_notifier() must have been previously called for this function
+ * usb_register_notify() must have been previously called for this function
  * to work properly.
  */
 void usb_unregister_notify(struct notifier_block *nb)
